cmd/armadactl/cmd/watch.go

```diff
@@ -53,6 +55,9 @@ var watchCmd = &cobra.Command{
 					summary += fmt.Sprintf(" | event: %s, job id: %s", reflect.TypeOf(e), e.GetJobId())
 					log.Info(summary)
 				}
+				if exit_on_inactive && !state.HasActiveJobs() {
+					os.Exit(0)
```

Comment:
I think if you just returned true here, it'd do the same and you wouldn't have to exit out of the application

Reply:
Done.